tests/functional/routes/test_tunnels.py

```diff
@@ -111,7 +111,7 @@ def test_tunnel_open_unowned_subdomain(self, client, current_user, session):
     def test_tunnel_close_owned(self, client, session, current_user):
         """User can close a tunnel"""
 
-        sub = subdomain.SubdomainFactory(user=current_user)
+        sub = subdomain.ReservedSubdomainFactory(user=current_user)
```

Comment:
Why'd this get changed?

Reply:
It isn't related to this PR but it was pointed out to me that the test is called `test_tunnel_close_owned` but it should have used the `ReservedSubdomainFactory` instead.  It would be more appropriate in a separate PR